change/@ni-nimble-components-248b2e34-5be4-492b-8512-1e5226e10ca4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Make the icon column fixed size and remove its fractional-width and min-pixel-width configuration",
```

Comment:
nit: could mention the new recommendation about header content for icon columns too

Reply:
This doesn't apply anymore to the change file description. The documentation for `width-mode` states that `iconSize` should only be used when the header and all mappings contain only icons.